plugins/config/src/ConfigurationEditorWidget/components/__snapshots__/ConfigurationEditor.test.js.snap

```diff
@@ -1303,7 +1303,7 @@ exports[`ConfigurationEditor widget renders with defaults of the PileupTrack sch
                 aria-invalid="false"
                 class="MuiInputBase-input MuiInput-input"
                 type="number"
-                value="600"
+                value="1200"
```

Comment:
What's causing this snapshot to update?

Reply:
not sure, i think it's something from master. colin committed the same snap 2 days ago on master with the message 'update snap', this was before I updated branch from master